### DISCUSSION TITLE
Pyg distributed benchmarks additional details

### DISCUSSION THREAD

**Discussion by denadai2** [pyg-team/pytorch_geometric#9640] [Q&A]
Dear Pytorch team,
could you share more details about this benchmark here https://pytorch-geometric.readthedocs.io/en/latest/tutorial/distributed_pyg.html#results-and-performance? E.g. how many epochs, num neighbors, whether you measured the time from the beginning of the script (thus including data loading) or just within the train loop

thanks!

**Comment by rusty1s** [comment]
@ZhengHongming888. I didn't do the benchmarks myself, but AFAIK it uses the default set of hparams in `distributed/pyg/node_ogb_cpu.py`, and only measures time within the training loop.
